# Análisis de la Encuesta Multiproposito

## Importación de Paquetes y carga del archivo

In [1]:
import pandas as pd
from matplotlib import pyplot as plt
import seaborn as sns
from scipy import stats
import numpy as np

In [2]:
data = pd.read_excel('Encuesta_Multiproposito_Suba.xlsx')

In [3]:
data.shape

(24536, 498)

**Hay en total 24536 encuestados en la Localidad de Suba**

# Onceava Sección (NPCLP)

## NHCLP2A
###	2. Durante los ÚLTIMOS 12 MESES, ¿de cuáles de los siguientes hechos ha sido víctima usted o alguna persona del hogar?:
1. Atracos o robos
2. Homicidios o asesinatos
3. Persecusión o amenazas contra la vida
4. Extorsión o chantaje
5. Acoso (sexual, laboral, escolar, psicológico, ciberacoso)


## NHCLP2AA
###	2a. ¿Entre 1985 y el momento actual, usted o algún miembro del hogar ha sido víctima del conflicto armado?	
1. Sí
2. No

**Datos: **

## NHCLP2AB
###	2a. 1. ¿Alguna de estas personas fue víctima de desplazamiento forzado?	
1. Sí
2. No

**Datos: **

## NHCLP2AC
###	2a. 2. ¿Alguna de estas personas se encuentra incluida en el Registro Único de Víctimas-RUV?	
1. Sí 
2. No
9. No sabe

**Datos: **

## NHCLP5
###	5. Usted piensa que el nivel de vida actual de su hogar, respecto al que tenía 5 años atrás es:	
1. Mejor
2. Igual
3. Peor

**Datos: **

## NHCLP7A	
### 7. Entre el  año 2017 y el momento actual ¿cree que  la ciudad ha mejorado, empeorado o se mantiene igual, en cada uno de los siguientes equipamientos?:
1. Vías y puentes vehiculares
2. Andenes
3. Puentes peatonales
4. Ciclorutas
5. Parques y zonas verdes
6. Hospitales y centros de salud
7. Colegios
8. Sistema Integrado de Transporte (Trasmilenio, alimentadores, SITP)
9. Escenarios artísticos y culturales
10. Escenarios deportivos
11. Zonas comerciales	

Opciones
1. Mejor
2. Igual
3. Peor
9. No sabe

**Datos: **

## NHCLP8A
###	8. Entre el  año 2017 y el momento actual ¿cree que  la ciudad ha mejorado, empeorado o se mantiene igual, en cada uno de los siguientes aspectos?: 
1. Atención en las oficinas públicas del distrito
2. Vigilancia, seguridad y reacción oportuna y eficiente de la policía
3. Educación pública 
4. Transporte público urbano
5. Programas de alimentación y nutrición
6. Respeto a los derechos humanos
7. Igualdad de oportunidades para las mujeres
8.  Igualdad de oportunidades para las personas mayores
9. Igualdad de oportunidades para la población en condición de discapacidad
10. Igualdad de oportunidades para los grupos LGBTI (Lesbianas, Gays, Bisexuales, Transexuales, Intersexuales)
11. Manera en que la Alcaldía de Bogotá comunica sus decisiones, acciones, resultados y oferta institucional
12. En la lucha contra la corrupción en la ciudad
13. Fortalecimiento de la participación ciudadana
14. Protección y garantía de la libertad de conciencia, religiosa y de cultos
Opciones
1. Mejor
2. Igual
3. Peor
9. No sabe

**Datos: **

## NHCLNP8C
###	8c. ¿De los siguientes aspectos cuál considera que es el más importante para que las entidades públicas mejoren su desempeño institucional?	
1. Estrategias de lucha contra la corrupción
2. Participación ciudadana
3. Mayor presencia de las entidades públicas en el territorio
4. Servidores competentes
5. Articulación entre el gobierno nacional y departamental
6. Otro

**Datos: **

## NHCLP9A
###	9. Entre el  año 2017 y el momento actual ¿cree que  la ciudad ha mejorado, empeorado o se mantiene igual, en cada  uno de los siguientes aspectos ambientales?:
1. Calidad del agua potable	
2. Calidad del aire
3. Disposición de basuras
4. Educación ambiental
5. Barrido y aseo de calles
6. Reciclaje de residuos

Opciones
1. Mejor
2. Igual
3. Peor
9. No sabe

**Datos: **

## NHCLP10
###	10. Los ingresos de su hogar:	
1. No alcanzan para cubrir los gastos mínimos
2. Solo alcanzan para cubrir los gastos mínimos
3. Cubren más que los gastos mínimos

**Datos: **

## NHCLP12E
###	12. ¿Durante los ÚLTIMOS 12 MESES, alguna persona de este hogar recibió ayudas o subsidios en dinero o en especie de entidades del gobierno nacional, departamental o municipal por concepto de Otro:
	
- 1. Sí
- 2. No

**Datos: **

## NHCLP12_1
### 	¿cuántos otros programas?

**Datos: **

## NHCLP12EB
### ¿Cuántos miembros del hogar recibieron el subsidio?

**Datos: **

## NHCLP12EC
###	¿Cuánto recibieron en los últimos 12 meses

**Datos: **

## NHCLP12EB2
###	¿Cuántos miembros del hogar recibieron el subsidio?

**Datos: **

## NHCLP12EC2
###	¿Cuánto recibieron en los últimos 12 meses

**Datos: **

## NHCLNP12
### 	12a. ¿Durante los últimos 12 meses este hogar recibió mercados en especie por parte del gobierno nacional, departamental o municipal?	
1. Sí
2. No

**Datos: **

## NHCLNP12A	
### 1. ¿En cuánto estima el valor de todos los mercados recibidos?

**Datos: **


## NHCLNP12B
###	2. ¿Estos mercados: Los reciben periódicamente?"	
1. Sí
2. No

**Datos: **

## NHCLP14
### 	13. ¿Por falta de dinero, alguna persona integrante del hogar no consumió ninguna de las tres comidas (desayuno, almuerzo, comida), uno o más días de la SEMANA PASADA? 	

1. Sí
2. No

**Datos: **

## NHCLP16
###	14. ¿En los ÚLTIMOS 30 DÍAS, por falta de dinero u otros recursos, ¿alguna vez usted se preocupó porque los alimentos se acabaran en su hogar?	
1. Sí
2. No

**Datos: **

## NHCLP17
### 	15. ¿En los ÚLTIMOS 30 DÍAS, por falta de dinero u otros recursos, ¿alguna vez en su hogar se quedaron sin alimentos?	
1. Sí
2. No

**Datos: **

## NHCLP20
###	18. ¿En los ÚLTIMOS 30 DÍAS, por falta de dinero u otros recursos, ¿alguna vez usted o algún adulto en su hogar dejó de desayunar, almorzar o cenar?	
1. Sí
2. No

**Datos: **

## NHCLP26
###	24. ¿En los ÚLTIMOS 30 DÍAS, por falta de dinero u otros recursos, ¿alguna vez algún menor de 18 años en su hogar dejó de desayunar, almorzar o cenar?	
1. Sí
2. No

**Datos: **

## NHCLP29_1A
###	29. A raíz de la pandemia del coronavirus (COVID-19), ¿cuáles de las siguientes medidas tomaron los miembros del hogar para hacer frente a los efectos económicos:
1. Uno o más miembros del hogar que no trabajaban empezaron a trabajar
2. Adoptaron nuevas fuente de ingreso
3. Cambiaron de vivienda
4. Gastaron parte o todos sus ahorros
5. Se endeudaron o ampliaron el plazo de alguna(s) deuda(s)"
6. Vendieron algunos bienes o activos (vivienda, moto, carro, electrodomésticos, lote)
7. Arrendaron o hipotecaron la vivienda ocupada por el hogar
8. Algún miembro del hogar tuvo que retirarse de la escuela, colegio, jardín, universidad o carrera técnica o tecnológica
9. Disminuyeron el gasto en alimentos
10. Pidieron ayuda de familiares, amigos u otras personas del barrio, pueblo o vereda
11. Solicitaron subsidio de desempleo
12. Ninguna de las anteriores"

**Datos: **

## NHCLP31AA
###	31. En su hogar ¿Quién? ¿Aporta dinero al hogar?
1. El/la jefe/a del hogar
2. El/la cónyuge
3. Otro(s) miembro(s) del hogar

**Datos: **

## NHCLP31BA
###	31. En su hogar ¿Quién? ¿Toma decisiones respecto al manejo del dinero?
1. El/la jefe/a del hogar
2. El/la cónyuge
3. Otro(s) miembro(s) del hogar

**Datos: **

## NHCLP31CA
###	31. En su hogar ¿Quién ¿Toma decisiones sobre dónde estudian los menores de edad? (Solo si hay menores)
1. El/la jefe/a del hogar
2. El/la cónyuge
3. 'Otro(s) miembro(s) del hogar

**Datos: **

In [ ]:
data.to_excel('Encuesta_Multiproposito_Suba.xlsx', index = False)